enhancements/console/quick-starts-crd.yaml

```diff
@@ -0,0 +1,173 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
I'm not sure we want to check in the CRD since it will drift from what we have in openshift/api over time. We can cover that during API review.